app/js/views/microphone.jsx

```diff
@@ -27,17 +27,56 @@ export default class Microphone extends React.Component {
     this.click = this.click.bind(this);
   }
 
+  loadAudio() {
+    const AudioContext = window.AudioContext || window.webkitAudioContext;
```

Comment:
We're supporting latest versions so I guess I can get rid of the `webkit` prefix.